contracts/0.4.24/Lido.sol

```diff
@@ -397,23 +418,6 @@ contract Lido is ILido, IsContract, StETH, AragonApp {
         emit RecoverToVault(vault, _token, balance);
     }
 
-    /**
-      * @notice Send NTFs to recovery Vault
-      * @param _token Token to be sent to recovery vault
-      * @param _tokenId Token Id
-      */
-    function transferERC721ToVault(address _token, uint256 _tokenId) external {
```

Comment:
I'm not sure, but think it's fine to live some time without capability to recover NFT.
Anyway it seems the next Lido.sol iteration gonna come quite soon and it will require refactoring lots of the code to a separate contract. And then there will be enough room to restore NTF recovering.

Reply:
Yep, I have talked with @ujenjt and we converge to the decision: if needed we would upgrade to recover some highly-priced-NFT stuff.